{{cookiecutter.project_slug}}/README.md

```diff
@@ -109,7 +109,7 @@ First, bring you project on the Mila cluster (assuming you didn't create your
 project directly there). To do so, simply login on the Mila cluster and git
 clone your project:
 
-    git@github.com:{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}.git
+    git clone git@github.com:{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}.git
```

Comment:
assumes git with ssh?

Reply:
yes, most likely (they can change it otherwise)